pkg/sdn/plugin/controller.go

```diff
@@ -232,64 +232,64 @@ func (plugin *OsdnNode) SetupSDN() (bool, error) {
-	otx.AddFlow("table=0, priority=200, in_port=1, ip, nw_src=%s, nw_dst=%s, actions=move:NXM_NX_TUN_ID[0..31]->NXM_NX_REG0[],goto_table:1", clusterNetworkCIDR, localSubnetCIDR)
+	otx.AddFlow("table=0, priority=200, in_port=1, arp, nw_src=%s, nw_dst=%s, actions=move:NXM_NX_TUN_ID[0..31]->NXM_NX_REG0[],goto_table:10", clusterNetworkCIDR, localSubnetCIDR)
+	otx.AddFlow("table=0, priority=200, in_port=1, ip, nw_src=%s, nw_dst=%s, actions=move:NXM_NX_TUN_ID[0..31]->NXM_NX_REG0[],goto_table:10", clusterNetworkCIDR, localSubnetCIDR)
 	otx.AddFlow("table=0, priority=150, in_port=1, actions=drop")
 	// tun0
-	otx.AddFlow("table=0, priority=200, in_port=2, arp, nw_src=%s, nw_dst=%s, actions=goto_table:5", localSubnetGateway, clusterNetworkCIDR)
-	otx.AddFlow("table=0, priority=200, in_port=2, ip, actions=goto_table:5")
+	otx.AddFlow("table=0, priority=200, in_port=2, arp, nw_src=%s, nw_dst=%s, actions=goto_table:30", localSubnetGateway, clusterNetworkCIDR)
+	otx.AddFlow("table=0, priority=200, in_port=2, ip, actions=goto_table:30")
 	otx.AddFlow("table=0, priority=150, in_port=2, actions=drop")
 	// else, from a container
-	otx.AddFlow("table=0, priority=100, arp, actions=goto_table:2")
-	otx.AddFlow("table=0, priority=100, ip, actions=goto_table:2")
+	otx.AddFlow("table=0, priority=100, arp, actions=goto_table:20")
+	otx.AddFlow("table=0, priority=100, ip, actions=goto_table:20")
 	otx.AddFlow("table=0, priority=0, actions=drop")
 
-	// Table 1: VXLAN ingress filtering; filled in by AddHostSubnetRules()
-	// eg, "table=1, priority=100, tun_src=${remote_node_ip}, actions=goto_table:5"
-	otx.AddFlow("table=1, priority=0, actions=drop")
+	// Table 10: VXLAN ingress filtering; filled in by AddHostSubnetRules()
+	// eg, "table=10, priority=100, tun_src=${remote_node_ip}, actions=goto_table:30"
+	otx.AddFlow("table=10, priority=0, actions=drop")
 
-	// Table 2: from OpenShift container; validate IP/MAC, assign tenant-id; filled in by openshift-sdn-ovs
-	// eg, "table=2, priority=100, in_port=${ovs_port}, arp, nw_src=${ipaddr}, arp_sha=${macaddr}, actions=load:${tenant_id}->NXM_NX_REG0[], goto_table:5"
-	//     "table=2, priority=100, in_port=${ovs_port}, ip, nw_src=${ipaddr}, actions=load:${tenant_id}->NXM_NX_REG0[], goto_table:3"
+	// Table 20: from OpenShift container; validate IP/MAC, assign tenant-id; filled in by openshift-sdn-ovs
+	// eg, "table=20, priority=100, in_port=${ovs_port}, arp, nw_src=${ipaddr}, arp_sha=${macaddr}, actions=load:${tenant_id}->NXM_NX_REG0[], goto_table:30"
+	//     "table=20, priority=100, in_port=${ovs_port}, ip, nw_src=${ipaddr}, actions=load:${tenant_id}->NXM_NX_REG0[], goto_table:30"
 	// (${tenant_id} is always 0 for single-tenant)
-	otx.AddFlow("table=2, priority=0, actions=drop")
-
-	// Table 3: from OpenShift container; service vs non-service
-	otx.AddFlow("table=3, priority=100, ip, nw_dst=%s, actions=goto_table:4", serviceNetworkCIDR)
-	otx.AddFlow("table=3, priority=0, actions=goto_table:5")
```

Comment:
We seem to be dropping table 3 completely, I can't find the mapping for that.  openshift-sdn-ovs should still refer to it in this patch, but it won't exist any more.